packages/nimble-tokens/.eslintignore

```diff
@@ -1,4 +1,3 @@
 dist/*
-!dist/styledictionary
 dist/styledictionary/*
-!dist/styledictionary/nimble-extensions.js
+source/styledictionary/build.js
```

Comment:
I fixed lint errors in most of the files that moved from `dist` to `source` but `build.js` had a parse error because of the way it used `import` and I didn't find an easy fix so left it unlinted.